app/internal/page/data.go

```diff
@@ -78,7 +81,14 @@ type Lpa struct {
 	CheckedAgain             bool
 	ConfirmFreeWill          bool
 	SignatureCode            string
-	IdentityOptions          []IdentityOption
+	IdentityOptions          IdentityOptions
+}
+
+type IdentityOptions struct {
+	Selected []IdentityOption
+	First    IdentityOption
+	Second   IdentityOption
+	Current  int
```

Comment:
probably would be better to store the results then confirm that the data is present to move on, also need to figure out what happens if you leave half-way through then start again from the task list

Reply:
yeah, going to change this so there are `FirstDone`/`SecondDone` flags that get set by the callback route (not the redirect), then can skip to the callback if the flag is set and the continue click won't double set

Reply:
and then can probably move the logic into handlers instead of redirecting multiple times